parent/pom.xml

```diff
@@ -1513,7 +1505,19 @@
                <buildOutputDirectory>${basedir}/eclipse-output</buildOutputDirectory>
             </configuration>
          </plugin>
-
+         <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-checkstyle-plugin</artifactId>
+            <executions>
+               <execution>
+                  <id>checkstyle</id>
+                  <phase>validate</phase>
```

Comment:
My rationale is if the code is not compiling, I'd like to get the compile error first instead of an eventual checkstyle error